doc/JSON_INFO.md

```diff
@@ -615,6 +615,103 @@ Every item type can have optional seed data, if the item has seed data, it's con
 }
 ```
 
+###ARTIFACT DATA
+Every item type can have optional artifact properties (which makes it an artifact):
+```JSON
+"artifact_data" : {
+    "charge_type": "ARTC_PAIN",
+    "effects_carried": ["AEP_INT_DOWN"],
+    "effects_wielded": ["AEP_DEX_UP"],
+    "effects_activated": ["AEA_BLOOD", "AEA_NOISE"],
+    "effects_worn": ["AEP_STR_UP"]
+}
+```
+
+#### charge_type
+(optional, default: "ARTC_NULL") How the item is recharged. For this to work, the item needs to be a tool that consumes charges upon invocation and has non-zero max_charges. Possible values (see src/artifact.h for an up-to-date list):
+- "ARTC_NULL" Never recharges!
+- "ARTC_TIME" Very slowly recharges with time
+- "ARTC_SOLAR" Recharges in sunlight
+- "ARTC_PAIN" Creates pain to recharge
+- "ARTC_HP" Drains HP to recharge
+
+#### effects_carried
+(optional, default: empty list) Effects of the artifact when it's in the inventory (main inventory, wielded, or worn) of the player. Possible values (see src/enums.h for an up-to-date list):
+- "AEP_STR_UP" Strength + 4
+- "AEP_DEX_UP" Dexterity + 4
+- "AEP_PER_UP" Perception + 4
+- "AEP_INT_UP" Intelligence + 4
+- "AEP_ALL_UP" All stats + 2
+- "AEP_SPEED_UP" +20 speed
+- "AEP_IODINE" Reduces radiation
+- "AEP_SNAKES" Summons friendly snakes when you're hit
+- "AEP_INVISIBLE" Makes you invisible
+- "AEP_CLAIRVOYANCE" See through walls
+- "AEP_SUPER_CLAIRVOYANCE" See through walls to a great distance
+- "AEP_STEALTH" Your steps are quieted
+- "AEP_EXTINGUISH" May extinguish nearby flames
+- "AEP_GLOW" Four-tile light source
+- "AEP_PSYSHIELD" Protection from stare attacks
+- "AEP_RESIST_ELECTRICITY" Protection from electricity
+- "AEP_CARRY_MORE" Increases carrying capacity by 200
+- "AEP_SAP_LIFE" Killing non-zombie monsters may heal you
```

Comment:
The description (copied from enums.h?) is wrong.
SAP_LIFE triggers on all melee attacks above 4 damage.